### ISSUE TITLE
Replace deprecated GtkTable with GtkGrid

### ISSUE THREAD

**Issue by elboulangero** [nicklan/pnmixer#149]
GtkTable is deprecated since Gtk 3.4, and somehow we forgot to get rid of it. It should be replaced by GtkGrid.

https://developer.gnome.org/gtk3/stable/GtkTable.html

**Comment by hasufell** [comment]
Whenever I try to edit our glade files and save them with the glade editor, stuff is missing again and I don't like manually editing huge xml files. Can we try to stay glade editor compliant? Why does it throw out stuff?

**Comment by elboulangero** [comment]
Yep it's a bit my mistake, at some point I tried to really get rid of everything deprecated, and edited the stuff manually. Now I realize it was the wrong approach, we should indeed be Glade compliant, and we shouldn't care when Glade generates some deprecated stuff...

Now I think it's easy to fix that by looking at the diff after saving with Glade. One of the main thing I remember is that I used the `margin` property on Widgets, and that's one of the thing wiped away by Glade. It should be replaced by `margin_start`, `margin_end`, `margin_top` and `margin_bottom` (yep, that's 4 properties instead of 1.

Apart from that, I think the rest is trivial.
